code/datums/components/plumbing/_plumbing.dm

```diff
@@ -208,12 +229,19 @@
 		net.add_plumber(src, dir)
 		net.add_plumber(P, opposite_dir)
 
+/datum/component/plumbing/proc/hide(atom/movable/AM, intact)
+
```

Comment:
weird spacing here. also why is the var named "intact"?

Reply:
Intact was always the name of the var. The hide element simply inherited the name.

I would rather keep it for compatibility reasons.